common/app/views/fragments/commercial/contentLogo.scala.html

```diff
@@ -1,11 +1,10 @@
 @(branding: common.commercial.Branding)
 
-<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge--interactive">
-    <div class="brandbadge__inner">
+<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge ad-slot--paid-for-badge--front">
```

Comment:
Can we use [this markup](https://github.com/guardian/frontend/blob/master/common/app/views/commercial/containerWrapper.scala.html#L29) here? Much simpler and also does not use `H3` which does not make sense.